tool/src/main/java/org/datacommons/tool/Processor.java

```diff
@@ -116,14 +122,41 @@ public static Integer process(Args args) throws IOException, TemplateException {
     return retVal;
   }
 
+  private void fetchNamesforSamplePlaces(Boolean verbose) throws IOException, InterruptedException {
```

Comment:
I wonder about moving this to ApiHelper as an API that given a list of DCIDs plus a property ("name" here), returns a map of DCID->value?

That way, we don't have such lower-level logic in Processor, and also move the "fetchNamesForSamplePlaces" callsite into StatsChecker?

Reply:
I moved the the logic to `StatChecker`. I support the idea of moving the logic of parsing the API results to a Map in ApiHelper, however, I believe that is out of the scope of this PR. Once this is merged, I can work on that (along with moving other DC API calls to ApiHelper) in a separate PR.